cmd/cql/internal/balance.go

```diff
@@ -0,0 +1,66 @@
+package internal
+
+import (
+	"strings"
+
+	"github.com/CovenantSQL/CovenantSQL/client"
+	"github.com/CovenantSQL/CovenantSQL/types"
+)
+
+var (
+	tokenName string // get specific token's balance of current account
+)
+
+var CmdBalance = &Command{
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var CmdBalance should have comment or be unexported